docker/buildandruncontainer.sh

```diff
@@ -0,0 +1,57 @@
+
+# Build blobfuse2 binary
+cd ..
+if [ "$1" == "fuse2" ]
```

Comment:
Should we also build the health monitor here?

Reply:
Just trying to have a very basic container which can mount blobfuse so avoiding all unnecessary stuff. If we combine health monitor here then config has to be setup for that and if we enable that by default then on container spawn it will be turned on which takes more memory and resources. So to conserve all that just keeping it very simple.